scripts/restore.sh

```diff
@@ -293,9 +293,10 @@ restore_shell_history() {
 
 restore_all_pane_processes() {
 	if restore_pane_processes_enabled; then
+		local save_command_strategy="$(get_tmux_option "$save_command_strategy_option" "$default_save_command_strategy")"
```

Comment:
Oh.  I forgot to remove that line after removing the dependency on `base64`.  There was an issue where it would potentially have broken other restore strategies if the command could be read as a base64 encoded string, which is why that line was included